src/main/java/net/eventsexpress/app/Common/BaseWrapper.java

```diff
@@ -0,0 +1,214 @@
+package net.eventsexpress.app.Common;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import org.apache.commons.configuration2.builder.fluent.Configurations;
+import org.apache.commons.configuration2.ex.ConfigurationException;
+import org.apache.commons.configuration2.Configuration;
+
+import java.io.File;
+import java.time.Duration;
+import java.util.List;
+
+
+public class BaseWrapper {
```

Comment:
Seems here you're going to mix Driver Configuration with Decorating of Selenium API. If so, it would be nice to split this class into two or more.